drracket/drracket/private/stick-figures.rkt

```diff
@@ -101,67 +101,63 @@
       (right-toe 109 132)))
   
   (define (get-size-parameters)
-    (let-values ([(min-rx min-ry) (get-max/min-x/y min running-points)]
-                 [(max-rx max-ry) (get-max/min-x/y max running-points)]
-                 [(min-wx min-wy) (get-max/min-x/y min waiting-points)]
-                 [(max-wx max-wy) (get-max/min-x/y max waiting-points)])
-      (let* ([running-w (* small-factor (- max-rx min-rx))]
-             [waiting-w (* small-factor (- max-wx min-wx))]
-             [running-h (* small-factor (- max-ry min-ry))]
-             [waiting-h (* small-factor (- max-wy min-wy))]
-             [w (+ 2 (ceiling (max running-w waiting-w)))]
-             [h (+ 2 (ceiling (max running-h waiting-h)))]
-             [running-dx (+ 1 (- (/ w 2) (/ running-w 2)))]
-             [running-dy (+ 1 (- (/ h 2) (/ running-h 2)))]
-             [waiting-dx (+ 1 (- (/ w 2) (/ waiting-w 2)))]
-             [waiting-dy (+ 1 (- (/ h 2) (/ waiting-h 2)))])
-        (values w h running-dx running-dy waiting-dx waiting-dy))))
+  (define-values (min-rx min-ry) (get-max/min-x/y min running-points))
```

Comment:
Why are these definitions not indented inside `get-size-parameters`? Or are they and I just don't see it for some reason?

Reply:
I think they are, GitHub is just wrapping the line

Reply:
This is what I see. There's no wrapping. 

<img width="603" alt="Screenshot 2023-04-01 at 6 30 37 PM" src="https://user-images.githubusercontent.com/9099577/229325883-ac9e4895-a6d9-4c14-a770-7b32f6723319.png">

Reply:
Me too.